### PR TITLE
Implement custom-resource-page example

### PR DESCRIPTION
**Pull request by nevalla**
Signed-off-by: Lauri Nevala <lauri.nevala@gmail.com>

### CODE REVIEW

**Review by jakolehm — COMMENTED** [review]
README is missing?

File: custom-resource-page/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "lens-crd-sample",
```
Comment (danielpacak) [inline custom-resource-page/package.json]: Is the "publisher" property required?

File: custom-resource-page/src/certificate-store.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+export const certificatesApi = new CertifcatesApi({
+  objectConstructor: Certificate
+});
+
+export class CertificatesStore extends K8sApi.KubeObjectStore<Certificate> {
+  api = certificatesApi
+}
+
+export const certificatesStore = new CertificatesStore();
+K8sApi.apiManager.registerStore(certificatesStore);
```
Comment (danielpacak) [inline custom-resource-page/src/certificate-store.ts:15]: I think this has changed in the latest API
```suggestion
K8sApi.apiManager.registerStore(certificatesStore,[certificatesApi]);
```

**Comment by nevalla** [comment]
> README is missing?

Added

File: custom-resource-page/src/certificate-store.ts (hunk 2 above)
Comment (nevalla) [reply custom-resource-page/src/certificate-store.ts:15]: api parameter is optional. If not provided any, api defined in store object will be used.

**Review by jakolehm — APPROVED** [review]

**Pull request merged**